### ISSUE TITLE
Missing space between quiet and spec parameters

### ISSUE THREAD

**Issue by andr3a88** [xavierLowmiller/xcodegen-action#4]
I noticed that there is a space missing between the parameters of the xcodegen command. this causes the action to fail.

`/usr/local/bin/xcodegen --quiet--spec project.yml`  

Full log from Github build:

````
Run xavierLowmiller/xcodegen-action@1.1.1
/usr/local/opt/curl/bin/curl --silent --create-dirs --location --output /Users/runner/action-xcodegen/xcodegen.zip https://github.com/yonaskolb/XcodeGen/releases/download/2.19.0/xcodegen.zip
/usr/bin/unzip -q -o /Users/runner/action-xcodegen/xcodegen.zip
/bin/rm /Users/runner/action-xcodegen/xcodegen.zip
/Users/runner/action-xcodegen/xcodegen/install.sh
/bin/rm -rf /Users/runner/action-xcodegen/
/usr/local/bin/xcodegen --quiet--spec project.yml
(node:925) UnhandledPromiseRejectionWarning: Error: The process '/usr/local/bin/xcodegen' failed with exit code 1

    at ExecState._setResult (/Users/runner/work/_actions/xavierLowmiller/xcodegen-action/1.1.1/node_modules/@actions/exec/lib/toolrunner.js:574:25)
Usage: xcodegen [options]
    at ExecState.CheckComplete (/Users/runner/work/_actions/xavierLowmiller/xcodegen-action/1.1.1/node_modules/@actions/exec/lib/toolrunner.js:557:18)

    at ChildProcess.<anonymous> (/Users/runner/work/_actions/xavierLowmiller/xcodegen-action/1.1.1/node_modules/@actions/exec/lib/toolrunner.js:451:27)
Generate an Xcode project from a spec
    at ChildProcess.emit (events.js:210:5)

    at maybeClose (internal/child_process.js:1021:16)
Options:
    at Socket.<anonymous> (internal/child_process.js:430:11)
      --cache-path <value>      Where the cache file will be loaded from and save to. Defaults to ~/.xcodegen/cache/{SPEC_PATH_HASH}
    at Socket.emit (events.js:210:5)
  -h, --help                    Show help information
    at Pipe.<anonymous> (net.js:659:12)
  -n, --no-env                  Disable environment variable expansions
(node:925) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
      --only-plists             Generate only plist files
(node:925) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
  -p, --project <value>         The path to the directory where the project should be generated. Defaults to the directory the spec is in. The filename is defined in the project spec
  -r, --project-root <value>    The path to the project root directory. Defaults to the directory containing the project spec.
  -q, --quiet                   Suppress all informational and success output
  -s, --spec <value>            The path to the project spec file. Defaults to project.yml
  -c, --use-cache               Use a cache for the xcodegen spec. This will prevent unnecessarily generating the project if nothing has changed

Error: unrecognized option '--quiet--spec project.yml'
````

**Comment by xavierLowmiller** [comment]
Hi Andrea, thanks for bringing this to my attention! I'll try to provide a fix later today.

Also, feel free to provide a PR :+1:.

**Comment by xavierLowmiller** [comment]
Hey @andr3a88,

Can you try again with the newly released `1.1.2` version? I've tried it with a private test project that has similar options to yours, and this should be supported now :+1:
